portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppLocalServiceImpl.java

```diff
@@ -88,6 +88,12 @@
  */
 public class DLAppLocalServiceImpl extends DLAppLocalServiceBaseImpl {
 
+	/**
```

Comment:
I prefer to not deprecate the methods. Can you send it without it and see if it passes? Forcing to pass a external code to people only using the services, that won't ever use it and force them to know what it is...

Reply:
Sure. I'll remove the commits deprecating the methods